src/ringmesh/basic/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ target_sources(${target_name}
         "${lib_source_dir}/geometry_position.cpp"
         "${lib_source_dir}/geometry.cpp"
         "${lib_source_dir}/nn_search.cpp"
```

Comment:
I haven't realized in the past that we have to write the file name of .cpp and .h files in the CMakeLists.txt files. Why do we not loop over the files in each RINGMesh lib directory?

Reply:
because using glob is not a good idea, even cmake doest not recommand using it: Note We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.

Reply:
https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak